azure/v26.0.0/release.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: azure-26.0.0
+spec:
+  apps:
+  - name: azuredisk-csi-driver
```

Comment:
@giantswarm/team-phoenix shall we update azuredisk-csi-driver it seems a bit outdated but I don't know what you plan is here.
We could easily upadate to [1.30.x](https://github.com/kubernetes-sigs/azuredisk-csi-driver?tab=readme-ov-file#container-images--kubernetes-compatibility)

Reply:
Opened a PR, https://github.com/giantswarm/azuredisk-csi-driver-app/pull/72